### PR TITLE
Reworks Green slime core blacklist to use a species flag.

### LINKED ISSUE #11879: Slime toxin
Can turn you into a plasma man and you die within 30 seconds, pretty shitty race to be turned into.

### PR DESCRIPTION
**Pull request by Shadowlight213**
Fixes #11879
Closes #13884 
This Pr removes the list system rather than adding the species to the list.
Shadowlings, plasmamen, and synths have been added to the blacklist as the new system has all the children of a species blacklisted.

### CODE REVIEW

**Comment by Incoming5643** [comment]
Aw man I loved freaking people out as like the first goddamn synth in the world

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -38,6 +38,7 @@
 	var/meat = /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human //What the species drops on gibbing
 	var/list/no_equip = list()	// slots the race can't equip stuff to
 	var/nojumpsuit = 0	// this is sorta... weird. it basically lets you equip stuff that usually needs jumpsuits without one, like belts and pockets and ids
+	var/blacklisted = null //Flag to exclude from green slime core species.
```
Comment (MrStonedOne) [inline code/modules/mob/living/carbon/human/species.dm]: might want to make this 0 as null implies it can hold other/text values

Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/species.dm]: I saw dangerous_existence was null so I assumed it should be null.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -549,6 +554,7 @@ var/global/list/synth_flesh_disguises = list()
 	SA_para_min = 0
 	SA_sleep_min = 0
 	dangerous_existence = 1
```
Comment (Incoming5643) [inline code/modules/mob/living/carbon/human/species_types.dm]: While you're here, could you remove dangerous_existence from synths. They were only added here to keep them blacklisted from stuff that you've now made a better var to use for that. It's not actually dangerous at all to be a synth

Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/species_types.dm]: K.

Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/species_types.dm]: @Incoming5643 actually, I'm going to keep this, as I've seen the synths not get their disguises built properly, leaving them to look really weird.

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species_types.dm]: Synths always get their disguises properly, I tested it, if you disagree give me a test case.

Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/species_types.dm]: @RemieRichards Said round on sybil where I showed you.

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species_types.dm]: yeah shutup

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/species_types.dm]: and technically I'm still right, if you actually attempt to give them a disguise, they always get it :P

**Comment by LatD** [comment]
Just adding the military synth to blacklisted should be enough in my opinion. Normal ones are pretty harmless and fun.

**Pull request merged**